specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/costmanagement.json

```diff
@@ -260,6 +260,242 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.CostManagement/forecast": {
+      "post": {
+        "tags": [
+          "Forecast"
+        ],
+        "operationId": "Forecast_List",
+        "description": "Lists the forecast charges for scope defined.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "BillingAccountForecast": {
+            "$ref": "./examples/BillingAccountForecast.json"
+          },
+          "BillingProfileForecast": {
+            "$ref": "./examples/BillingProfileForecast.json"
+          },
+          "InvoiceSectionForecast": {
+            "$ref": "./examples/InvoiceSectionForecast.json"
+          },
+          "EnrollmentAccountForecast": {
+            "$ref": "./examples/EnrollmentAccountForecast.json"
+          },
+          "DepartmentForecast": {
+            "$ref": "./examples/DepartmentForecast.json"
+          },
+          "SubscriptionForecast": {
+            "$ref": "./examples/SubscriptionForecast.json"
+          },
+          "ResourceGroupForecast": {
+            "$ref": "./examples/ResourceGroupForecast.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "$filter",
+            "description": "May be used to filter forecasts by properties/usageDate (Utc time), properties/chargeType or properties/grain. The filter supports 'eq', 'lt', 'gt', 'le', 'ge', and 'and'. It does not currently support 'ne', 'or', or 'not'.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/scopeForecastParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ForecastDefinition"
+            },
+            "description": "Parameters supplied to the CreateOrUpdate Forecast Config operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/QueryResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.CostManagement/{externalCloudProviderType}/{externalCloudProviderId}/forecast": {
+      "post": {
```

Comment:
>post [](start = 7, length = 4)

Why is the list operation implemented using POST? Is it because you're supporting OData?

Reply:
same as above